src/pages/view/FilterBar/FilterBadge.js

```diff
@@ -1,74 +1,73 @@
 import { useDhis2ConnectionStatus } from '@dhis2/app-runtime'
 import i18n from '@dhis2/d2-i18n'
-import { Tooltip } from '@dhis2/ui'
+import { IconCross16, Tooltip } from '@dhis2/ui'
 import cx from 'classnames'
 import PropTypes from 'prop-types'
 import React from 'react'
 import { connect } from 'react-redux'
 import { acSetActiveModalDimension } from '../../../actions/activeModalDimension.js'
-import { useCacheableSection } from '../../../modules/useCacheableSection.js'
+import { useWindowDimensions } from '../../../components/WindowDimensionsProvider.js'
 import { sGetSelectedId } from '../../../reducers/selected.js'
 import classes from './styles/FilterBadge.module.css'
 
-const FilterBadge = ({ dashboardId, filter, openFilterModal, onRemove }) => {
-    const { isConnected: online } = useDhis2ConnectionStatus()
-    const { isCached } = useCacheableSection(dashboardId)
+const getFilterValuesText = (values) =>
+    values.length === 1
+        ? values[0].name
+        : i18n.t('{{count}} selected', { count: values.length })
```

Comment:
I wonder if we should handle plurals here, even though singular and plural are the same in this case? Might it be different in other languages?

Reply:
Yes, but some languages have multiple plural forms depending on how many things are being counted.

Reply:
AH OK, in that case we do have to add something like that. Could you advise on a change, since I am not familiar with this?